docker-compose.yml

```diff
@@ -151,10 +120,10 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock
     depends_on:
       - backend
-      - backend-ropsten
-      - backend-rinkeby
+      - backend-arbitrum-one
       - backend-goerli
+      - backend-rinkarby
```

Comment:
I am a bit puzzled by `rinkarby` -- ist that some pun about `rinkeby + arbitrum` or just a plain missspeling?

Reply:
got it, it's indeed what I suspected.